ui/src/app/constants/texts.constants.ts

```diff
@@ -69,6 +69,7 @@ export const texts = {
   UPDATE_WORKFLOW_CONFIRMATION_CONTENT: 'Are you sure you want to update the workflow?',
   UPDATE_WORKFLOW_SUCCESS_NOTIFICATION: 'Workflow has been updated.',
   UPDATE_WORKFLOW_FAILURE_NOTIFICATION: 'Sorry, something went wrong. Try again.',
+  UPDATE_WORKFLOW_OPTIMISTIC_LOCKING_FAILURE_NOTIFICATION: 'Sorry, workflow was updated in the meantime. Please refresh and try again.',
```

Comment:
How about "Please refresh and apply your changes again"? Users might simply hit refresh and assume that their changes will somehow stay and thus lose all of their changes

Reply:
Done